### ISSUE TITLE
Disable/Enable virtual monitors

### ISSUE THREAD

**Issue by joaoewald** [phillipberndt/fakexrandr#56]
There is an way to enable and disable virtual monitor without remove the configuration?
I'm currently running 3 virtual monitors on 4k tv while using as pc monitor (study and keyboard/mouse gaming). When I play rocket league or another "controller friendly" game, want to use all 4k resolution (single monitor), so I would like to easily switch between single monitor and fakexrandr 3 virtual monitors.
![virtual_monitos](https://user-images.githubusercontent.com/25589948/59511067-b92f6480-8e8b-11e9-826b-bcc82b15d689.png)

**Comment by phillipberndt** [comment]
> There is an way to enable and disable virtual monitor without remove the configuration?

No, you'd have to add support for that to fakexrandr explicitly. As you might have noticed by my late reply, I don't really maintain this library much anymore. I'll mark this as "help wanted".
